cartridge/utils.lua

```diff
@@ -356,13 +356,21 @@ local function http_read_body(req)
     -- that have a special meaning with % to escape them.
     -- A list of special characters is ().+-*?[]^$%
     local boundary_line = string.gsub('--'..boundary, "[%(%)%.%+%-%*%?%[%]%^%$%%]", "%%%1")
-    local _, form_body = req_body:match(
+    local payload_headers, form_body = req_body:match(
         boundary_line .. '\r\n' ..
         '(.-\r\n)' .. '\r\n' .. -- headers
         '(.-)' .. '\r\n' .. -- body
         boundary_line
     )
-    return form_body
+
+    local filename, _
```

Comment:
```suggestion
    local filename
```

Reply:
No. Otherwise _ will be considered as undefined variable